### PR TITLE
[EOC-20] Update colors to Sandstream palette - move to other branch

### PR DESCRIPTION
**Pull request by adamklepacz**

### CODE REVIEW

File: src/client/components/Header.js [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import React from 'react';
 
 import AppIcon from '../assets/images/coffee-solid.svg';
-import {
-  COMPANY_NAME,
-  COMPANY_PAGE_URL,
-  PROJECT_NAME
-} from '../common/variables';
+import { COMPANY_PAGE_URL, PROJECT_NAME } from '../common/variables';
+import SandstreamLogo from '../assets/images/sandstream_logo.png';
```
Comment (marekrozmus) [inline src/client/components/Header.js]: please change to something more general like CompanyLogo and also change file name

File: src/client/components/Header.js [hunk 2]
```diff
@@ -19,7 +16,11 @@ const Header = props => (
       rel="noopener noreferrer"
       target="_blank"
     >
-      {COMPANY_NAME}
+      <img
+        alt="Sandstream logo"
```
Comment (marekrozmus) [inline src/client/components/Header.js]: please change to "Company logo"

File: src/client/styles/variables/_colors.scss [hunk 3]
```diff
@@ -1,17 +1,20 @@
 $green: #08ab6b;
 $light-green: #e1f5c4;
-$purple: #5562eb;
 $red: #fa0202;
 $red-light: #e0a5a5;
 $gray1: #646c73;
```
Comment (alemikolo) [inline src/client/styles/variables/_colors.scss]: Color $gray1: #646c73 (used to text-normal) is similar to rgb(65, 64, 66)/#414042; which is defined in Sandstream colors book. It's a subtle change, but maybe it's better to use it with a stencil?

Comment (adamklepacz) [reply src/client/styles/variables/_colors.scss]: I've tried and it's much more darker. So for now I leave it as it is.

**Pull request merged**